src/ui/public/kbn_share/kbn_share.less

```diff
@@ -0,0 +1,42 @@
+.sharing-ui-menu__List {
+  display: block;
+  margin: 0;
+  padding: 0;
+}
+
+.sharing-ui-menu__ListWrapper,
+.sharing-ui-menu__SharePanel {
```

Comment:
I didn't notice this earlier, but sharing a styles declaration really tightly couples these two classes. This is great if there is some commonality or cohesion from the perspective of their role as UI components. But if it's just coincidence that `.sharing-ui-menu__ListWrapper` and `.sharing-ui-menu__SharePanel` share styles then I think it's better to duplicate the styles and split the classes apart. I think this will improve maintainability because then they can be changed independently without affecting one another.